Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/logicalStructure.xhtml

```diff
@@ -154,7 +162,7 @@
                 <f:actionListener binding="#{DataEditorForm.addDocStrucTypeDialog.preparePreselectedViews()}"/>
             </p:menuitem>
             <p:menuitem value="#{msgs.changeDocstructType}"
-                        rendered="#{DataEditorForm.structurePanel.selectedLogicalNode.data.dataObject['class'].simpleName ne 'View' and not DataEditorForm.structurePanel.selectedLogicalNode.data.linked}"
+                        rendered="#{DataEditorForm.structurePanel.selectedLogicalNode.type eq 'Structure' and not DataEditorForm.structurePanel.selectedLogicalNode.data.linked}"
```

Comment:
Maybe you can use the `StructurePanel.STRUCTURE_NODE_TYPE` constant here as well? (if it does not mean another request to the server!)